### ISSUE TITLE
bug: Emoji not properly decoding from packet

### ISSUE THREAD

**Issue by WouterK12** [PassTheMayo/minecraft-server-util#28]
**Describe the bug:**
Servers' motds can have emoji in them. Though, at the moment, they appear like this (marked yellow):
Discord:
![image](https://user-images.githubusercontent.com/34215333/101408856-4d418700-38dd-11eb-9e00-20c65e863068.png)
Postman:
![image](https://user-images.githubusercontent.com/34215333/101412071-410bf880-38e2-11eb-9f45-730c91347a7d.png)
Visual Studio Code terminal:
![image](https://user-images.githubusercontent.com/34215333/101412189-76184b00-38e2-11eb-827e-e0da6a1c2214.png)

I think this has something to do with reading the begin and end of a Byte from the buffer.

**Expected behavior:**
If a server has an motd with an emoji like this one: ⛄,
it should be decoded properly and display in the response.

**Additional context**
I did some debugging and found that all data from the buffer has no numbers with more than three digits. Emoji mostly have more than three digits before using `String.fromCodePoint()`.
```js
console.log("⛄".codePointAt(0));
// 9924
console.log(String.fromCodePoint(9924));
// ⛄
```
Maybe the problem is in `readVarInt()`? Since that's where the length of a Byte is determined?
Thanks in advance!

**Comment by PassTheMayo** [comment]
I noticed this a while ago too but wasn't able to find a fix. The culprit is how I'm reading the binary string from the socket during the `Response 0x00` packet. I'll look into your solution sometime this week.

https://github.com/PassTheMayo/minecraft-server-util/blob/master/src/structure/Packet.ts#L367

**Comment by PassTheMayo** [comment]
This bug has been fixed in the new version `3.3.0`. Please update and verify that the bug has been fixed or I will close this issue.

**Comment by WouterK12** [comment]
Before:
![image](https://user-images.githubusercontent.com/34215333/101604919-d9e56580-3a01-11eb-843a-5539174f1946.png)

After (`3.3.1`):
![image](https://user-images.githubusercontent.com/34215333/101605028-fa152480-3a01-11eb-8980-b3e4ba9c7aa3.png)

Yes, everything seems to be decoding properly now.
Thank you so much! 😄
